docs/src/ci-intro-js.md

```diff
@@ -1,19 +1,22 @@
 ---
 id: ci-intro
 title: "CI Github Actions"
+tags:
+  - Getting started
+  - Devops
 ---
 
 When installing Playwright you are given the option to add a [GitHub Actions](https://docs.github.com/en/actions). This creates a `playwright.yml` file inside a `.github/workflows` folder containing everything you need so that your tests run on each push and pull request into the main/master branch.
 
 **What you will learn:**
 
-- [How to use GitHub Actions to run your tests](#github-actions)
-- [How to create a repo and push to GitHub](#create-a-repo-and-push-to-github)
-- [How to open the workflows](#opening-the-workflows)
-- [How to view the test logs](#viewing-test-logs)
-- [How to download the report from GitHub](#downloading-the-playwright-report)
-- [How to view the report](#viewing-the-playwright-report)
-- [How to view the trace](#viewing-the-trace)
+- [How to use GitHub Actions to run your tests](/intro.md#github-actions)
```

Comment:
whats the reason for that change?

Reply:
if you don't do this docusauras will treat it as a TOC and add the whats next section and update it so the only way to trick it is to add that change.